ONI-DenseLogic/DenseLogicStrings.cs

```diff
@@ -37,6 +37,19 @@ public static class DENSELOGICTEAM_DENSEGATE {
 					public static LocString PORTOUT_INACTIVE = $"Writes {UI.FormatAsAutomationState("Red", UI.AutomationState.Standby)} signals to Automation Ribbons.";
 				}
 
+				public static class DENSELOGICTEAM_LOGICSEVENSEGMENT {
+					public static LocString NAME = UI.FormatAsLink("Seven Segment Display",
+						LogicSevenSegmentConfig.ID);
+					public static LocString DESC = $"Giving the duplicants a way to show off their favorite numbers has never been easier.";
+					public static LocString EFFECT = $"Displays the value of a {UI.FormatAsLink(STRINGS.BUILDINGS.PREFABS.LOGICRIBBON.NAME, LogicRibbonConfig.ID)} as a number 0 through 9.";
+					public static LocString LOGIC_PORT = "Numerical Input";
+					public static LocString INPUT_PORT_ACTIVE = $"Display a non-zero number.";
+					public static LocString INPUT_PORT_INACTIVE = $"Display zero.";
```

Comment:
Should be "Displays zero."

Reply:
yes I agree with asquared. I'll change it.